src/Tizen.NUI/src/internal/WebView/WebHitTest.cs

```diff
@@ -0,0 +1,260 @@
+            Editable = 1 << 6,
+
+            /// <summary>
+            /// the area which is text
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Text = 1 << 7,
+        }
+
+        /// <summary>
+        /// Gets the context of the hit test.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ResultContext TestResultContext
+        {
+            get
+            {
+                return (ResultContext)Interop.WebHitTest.GetResultContext(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the link uri of the hit test.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string LinkUri
+        {
+            get
+            {
+                return Interop.WebHitTest.GetLinkUri(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the link title of the hit test.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string LinkTitle
+        {
+            get
+            {
+                return Interop.WebHitTest.GetLinkTitle(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the link label of the hit test.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string LinkLabel
+        {
+            get
+            {
+                return Interop.WebHitTest.GetLinkLabel(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the image uri of the hit test.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string ImageUri
+        {
+            get
+            {
+                return Interop.WebHitTest.GetImageUri(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the media uri of the hit test.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string MediaUri
+        {
+            get
+            {
+                return Interop.WebHitTest.GetMediaUri(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the tag name of hit element of the hit test.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string TagName
+        {
+            get
+            {
+                return Interop.WebHitTest.GetTagName(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the node value of hit element of the hit test.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string NodeValue
+        {
+            get
+            {
+                return Interop.WebHitTest.GetNodeValue(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the attribute data of hit element of the hit test.
```

Comment:
As far as I know, `PropertyMap` is key-value pairs.
Can you specify possible keys?

Reply:
Done.
It is impossible to list all w3c html attributes.
I just gave an example here.

Reply:
Thank you! The example is helpful.